flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js

```diff
@@ -19,10 +19,23 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
         this.sentMessages = [];
         this.activeSystem = 'FMGC';
         this.messageQueue = [];
+        this.inFocus = false;
+        this.lastInput = 0;
     }
     get templateID() {
         return "A320_Neo_CDU";
     }
+    get isInteractive() {
+        const mcduInput = NXDataStore.get("MCDU_KB_INPUT", "NONE");
+        switch (mcduInput) {
+            case "ALL":
+                return true;
+            case "RALT":
+                return false;
+            default:
+                return false;
+        }
+    }
```

Comment:
Cant this be simplified?

```suggestion
    get isInteractive() {
        return NXDataStore.get("MCDU_KB_INPUT", "NONE") === "ALL";
    }
```

Reply:
Leftover fragment from before the change, good catch